package.json

```diff
@@ -25,7 +25,9 @@
     "terser-webpack-plugin": "2.2.1",
     "updates": "9.3.0",
     "webpack": "4.41.2",
-    "webpack-cli": "3.3.10"
+    "webpack-cli": "3.3.10",
+    "fomantic-ui": "2.8.3",
+    "gulp": "4.0.2"
```

Comment:
We can probably do without gulp as a direct dependency as fomantic-ui already depends on it, guaranteeing it to be there.